Makefile

```diff
@@ -261,14 +261,14 @@ package_firmware:
 
 clean:
 	@rm -rf build_*/
-	@$(MAKE) -C NuttX/nuttx clean
+	-@$(MAKE) -C NuttX/nuttx clean
 
 submodulesclean:
 	@git submodule sync --recursive
 	@git submodule deinit -f .
 	@git submodule update --init --recursive --force
 
-distclean: submodulesclean
+distclean: submodulesclean clean
```

Comment:
The git clean below wipes out everything except the eclipse project files. Unless it isn't for you?